modules/azure-gaming/game-dev-vmss/main.bicep

```diff
@@ -235,6 +241,35 @@ resource vmss 'Microsoft.Compute/virtualMachineScaleSets@2021-04-01' = {
           provisionVMAgent: true
         }	
       }
+      extensionProfile: {
+        extensions: (enableAnalyticsDashboard ? [
+          {
+            name: 'MMAExtension'
+            properties: {
+              publisher: 'Microsoft.EnterpriseCloud.Monitoring'
+              type: 'MicrosoftMonitoringAgent'
+              typeHandlerVersion: '1.0'
+              autoUpgradeMinorVersion: true
+              settings: {
+                workspaceId: reference(analyticsWorkspaceID, '2015-03-20').customerId
```

Comment:
it would be more idiomatic to get a reference to the log analytics workspace with `existing` then getting properties from it.

```bicep
resource la 'Microsoft.OperationalInsights/workspaces@2021-06-01' existing = {
  name: 'foo' // if you need you can use split() with the resourceId
}

var workspaceId = la.properties.customerId
var primarySharedKey = la.listKeys().primarySharedKey
```